packages/cli/test/integration/generators/relation.has-one.integration.js

```diff
@@ -363,19 +316,13 @@ describe('lb4 relation HasOne', /** @this {Mocha.Suite} */ function () {
       },
       {
         relationType: 'hasOne',
-        sourceModel: 'CustomerClass',
-        destinationModel: 'AddressClass',
-        registerInclusionResolver: true,
-      },
-      {
-        relationType: 'hasOne',
-        sourceModel: 'CustomerClassType',
-        destinationModel: 'AddressClassType',
+        sourceModel: 'Customer',
+        destinationModel: 'Address',
         registerInclusionResolver: false,
       },
     ];
 
-    const sourceClassnames = ['Customer', 'CustomerClass', 'CustomerClassType'];
+    const sourceClassnames = ['Customer', 'Customer'];
```

Comment:
hmm why there are two 'Customer's? Also, line 347 removes the index `i`, which implies only one `sourceClassName` is needed I think.

Reply:
👍 You're right, I messed it up with `sourceFileName`. Fixed.